src/test/java/uk/gov/pay/card/service/CardServiceTest.java

```diff
@@ -1,39 +1,17 @@
 package uk.gov.pay.card.service;
 
-import org.junit.Before;
 import org.junit.Test;
-import uk.gov.pay.card.db.CardInformationStore;
-import uk.gov.pay.card.model.CardInformation;
-
-import java.util.Optional;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static uk.gov.pay.card.service.CardService.firstNCharacters;
 
 public class CardServiceTest {
-
-    private final CardInformationStore cardInformationStore = mock(CardInformationStore.class);
-
-    private CardService cardService;
-
-    @Before
-    public void setup() {
-        cardService = new CardService(cardInformationStore);
-    }
-
     @Test
-    public void shouldStripTheCardNumberTo11DigitsForBinRangeLookup() {
-
-        CardInformation expectedCardInformation = new CardInformation("visa", "D", "visa", 11000L, 13000L);
-        when(cardInformationStore.find("12345678901")).thenReturn(Optional.of(expectedCardInformation));
-
-        Optional<CardInformation> cardInformation = cardService.getCardInformation("1234567890123456");
-
-        assertTrue(cardInformation.isPresent());
-        assertEquals(expectedCardInformation, cardInformation.get());
-        verify(cardInformationStore).find("12345678901");
+    public void testFirstNCharacters() {
+        assertEquals("", firstNCharacters("ab", 0));
+        assertEquals("a", firstNCharacters("ab", 1));
+        assertEquals("ab", firstNCharacters("ab", 2));
+        assertEquals("ab", firstNCharacters("ab", 3));
+        assertEquals("1234567890", firstNCharacters("1234567890123456789", 10));
```

Comment:
This could be tested by mocking the `CardInformationStore` and checking that `CardService` passes the correct substring to it (which would mean `firstNCharacters(…)` wouldn’t need to be accessible).